dashboard/src/reducers/cluster.ts

```diff
@@ -95,10 +95,6 @@ const clusterReducer = (
         },
       };
     case getType(actions.namespace.errorNamespaces):
-      // Ignore error listing namespaces since those are expected
```

Comment:
I love seeing code like this removed :P (ignoring errors because they're expected, as it inevitably leads to debugging issues later).